### PR TITLE
Modify Rule S3363: Add VB.NET description

### PR DESCRIPTION
**Pull request by zsolt-kolbay-sonarsource**
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone
- [ ] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

File: rules/S3363/vbnet/rule.adoc [hunk 1]
```diff
@@ -0,0 +1,58 @@
+include::../introduction-dotnet.adoc[]
```
Comment (cristian-ambrosini-sonarsource) [inline rules/S3363/vbnet/rule.adoc:1]: As discussed via Slack, I suggest moving the list from the `Introduction` to the `Why is this an issue` section, it looks a bit weird visually

File: rules/S3363/csharp/rule.adoc [hunk 2]
```diff
@@ -13,7 +13,6 @@ include::../how-dotnet.adoc[]
 internal class Account
 {
     public DateTime Id { get; set; }
-
```
Comment (cristian-ambrosini-sonarsource) [inline rules/S3363/csharp/rule.adoc:16]: Opinion: I think this empty line separates a bit the Key highlighting what the rule is focusing on

File: rules/S3363/csharp/rule.adoc [hunk 3]
```diff
@@ -54,8 +51,8 @@ internal class Person
 ----
 internal class Person
 {
+    [Key]
```
Comment (cristian-ambrosini-sonarsource) [inline rules/S3363/csharp/rule.adoc:54]: Why? It should work even without the attribute

**Review by cristian-ambrosini-sonarsource — CHANGES_REQUESTED** [review]

File: rules/S3363/csharp/rule.adoc (hunk 3 above)
Comment (zsolt-kolbay-sonarsource) [reply rules/S3363/csharp/rule.adoc:57]: I renamed the property to `PersonIdentifier`. Now it requires the `[Key]` attribute.

**Review by cristian-ambrosini-sonarsource — APPROVED** [review]
LGTM

**Review by csaba-sagi-sonarsource — APPROVED** [review]
LGTM!

**Pull request merged**